app/controllers/comments_controller.rb

```diff
@@ -0,0 +1,15 @@
+class CommentsController < ApplicationController
+  def create
+    @job = Job.find(params[:job_id])
+    @comment = @job.comments.create(comment_params)
+
```

Comment:
might want to have an `if @comment.save` conditional in case the comment body is blank or something